src/aria-snapshots/menu-test-aria-ts-glide-core-menu-button-disabled.yml

```diff
@@ -1,5 +1,5 @@
-- button "Target"
-- menu "Target":
+- button "Toggle"
```

Comment:
Renamed it to "Toggle" so it's aligned with Modal's "Open" button and Drawer's "Toggle" button. 

Menu is a bit different from those two in that its button goes into its "target" slot. But I thought "Toggle" would still make more sense from the Storybook user's perspective. Let me know what you think.